packages/manager/modules/octavia-load-balancer/src/create/service.js

```diff
@@ -39,4 +42,73 @@ export default class OctaviaLoadBalancerCreateService {
       )
       .then(({ data }) => data);
   }
+
+  createLoadBalancer(
+    projectId,
+    size,
+    regionName,
+    privateNetwork,
+    subnet,
+    gateway,
+    // listeners,
+    name,
+  ) {
+    const networkInformation = {
+      networkId: privateNetwork.id,
+      subnetId: subnet.id,
+    };
+
+    if (!gateway?.length) {
```

Comment:
**question:** Checking for `gateway.length` but assigning an object. Should it be an object or an array?

Reply:
If there is no gateways, I should create 1 as a object. So this is correct